src/main/java/slimeknights/tconstruct/library/client/GuiUtil.java

```diff
@@ -154,8 +154,18 @@ private static FluidStack getFluidHovered(SmelteryTank tank, int y, int height)
     return null;
   }
 
+  private static int minFluidHeight = 3;
+
+  /**
+   * Sets the preference from the config. For internal use only
+   * @param minHeight  integer minimum display height for fluids
+   */
+  public static void setMinFluidHeight(int minHeight) {
```

Comment:
This class only exists on the client side, while the config loading is called don both sides. It will probably crash if you try running this on a server. It's probably going to be messy, but the best bet might be moving this to the general util file just so the setter is safe on a server.

Reply:
Good catch! Things got a little hectic as I was wrapping that up and testing on a sever slipped my mind. I moved `minFluidHeight` into `library/Util.java` and gave it a getter as well as the setter. Built and tested with server and client.